app/components/live-header.hbs

```diff
@@ -0,0 +1,13 @@
+<Tabs
+  data-test-live='header'
+  class='live__header'
```

Comment:
Isn't is seems little confusive that we are adding class `live__header` to Tabs

Reply:
We can keep `live__header-tab`. What do you think?

Reply:
Let's keep this general so it can be reused in other things if needed in future.

.tab will be good.

Reply:
I was thinking generic in the sense that if someone wants to reuse tabs they can style accordingly to their need. But I got your point.